src/main/java/com/aws/greengrass/componentmanager/plugins/docker/DockerImageDownloader.java

```diff
@@ -61,9 +62,16 @@ public class DockerImageDownloader extends ArtifactDownloader {
     private RetryUtils.RetryConfig finiteAttemptsRetryConfig =
             RetryUtils.RetryConfig.builder().initialRetryInterval(Duration.ofSeconds(10L))
                     .maxRetryInterval(Duration.ofMinutes(32L)).maxAttempt(30).retryableExceptions(
-                    Arrays.asList(DockerServiceUnavailableException.class, DockerLoginException.class,
+                            Arrays.asList(DockerServiceUnavailableException.class, DockerLoginException.class,
                             SdkClientException.class, ServerException.class)).build();
 
+    @Setter(AccessLevel.PACKAGE)
+    private RetryUtils.RetryConfig finiteDeleteAttemptsRetryConfig =
+            RetryUtils.RetryConfig.builder().initialRetryInterval(Duration.ofSeconds(10L))
+                    .maxRetryInterval(Duration.ofMinutes(2L)).maxAttempt(5).retryableExceptions(
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

These configs are flagged under GS-Tech Static config campaign and is considered to be a violation. Please migrate these configs